crates/package-manager/src/install_without_lockfile.rs

```diff
@@ -9,6 +10,8 @@ use pacquet_registry::PackageVersion;
 use pacquet_tarball::MemCache;
 use pipe_trait::Pipe;
 
+pub type ResolvedPackages = MemoMap<String, bool>;
```

Comment:
Please document what is the key and what is the value, what does `true` mean, and what does `false` mean.